bazel/repository_locations.bzl

```diff
@@ -303,4 +303,19 @@ REPOSITORY_LOCATIONS = dict(
         # 2019-11-19
         urls = ["https://github.com/protocolbuffers/upb/archive/8a3ae1ef3e3e3f26b45dec735c5776737fc7247f.tar.gz"],
     ),
+    kafka_source = dict(
+        sha256 = "ae7a1696c0a0302b43c5b21e515c37e6ecd365941f68a510a7e442eebddf39a1",  # 2.2.0-rc2
+        strip_prefix = "kafka-2.2.0-rc2/clients/src/main/resources/common/message",
+        urls = ["https://github.com/apache/kafka/archive/2.2.0-rc2.zip"],
```

Comment:
Congrats on shipping this @adamkotwasinski. I suspect this version lines up with when you started the PR. Kafka is now at 2.4.0 - https://github.com/apache/kafka/releases/tag/2.4.0. Are there any blockers on moving from the old RC release to the latest stable release?

Reply:
@moderation there should be none - I'll take a look as soon as I can

Reply:
@moderation work will be in https://github.com/envoyproxy/envoy/pull/9582 ; I will need some more time to get the 2.4 running (the descriptor files used to generate the C++ code changed a little)